R/data_interface.R

```diff
@@ -1,4 +1,5 @@
 spark_csv_options <- function(header,
```

Comment:
There are two other calls to `spark_csv_options` that require to pass `TRUE` as the second parameter.